project/ws/app/src/lib/routes/karma-programs/karma-programs-microsite/karma-programs-microsite.component.html

```diff
@@ -64,6 +71,14 @@ <h1 class=" txt-ellipsis-2 break-words">{{data?.title}}</h1>
                     {{data?.description | slice:0:descriptionMaxLength}}
                     <span *ngIf="data?.description?.length > descriptionMaxLength">...</span>
                 </p>
+                <ng-container *ngIf="data?.referenceLinks && data?.referenceLinks?.length">
+                    <ng-container *ngFor="let ref of data?.referenceLinks">
+                        <p >
```

Comment:
Is there any change we have to do for multi-lingual here?

Reply:
we can't do here bcz the values is coming from form json